### PR TITLE
removed unused parse_for_orders method from cart model

### PR DESCRIPTION
**Pull request by jimszalew**
LIST OF CHANGES:
removed parse_for_orders method from cart model, wasn't being used anywhere in the program

RSPEC GREEN?
YES

### CODE REVIEW

**Comment by jimszalew** [comment]
Removes unused cart model method 'parse_for_orders'. It wasn't used anywhere in the project.

File: app/models/cart.rb [hunk 1]
```diff
@@ -45,17 +45,6 @@ def remove_item(item_id)
...
-      price = Item.find(item_id).price
-      {
-        item_id: id.to_i,
-        quantity: quantity,
-        line_item_total: quantity * price
-      }
-    end
-  end
-
   def order_hash
```
Comment (mimizotti) [inline app/models/cart.rb:48]: Good catch, @jimszalew

**Review by mimizotti — COMMENTED** [review]
This looks good to go.

**Review by vaidashi — APPROVED** [review]
nice, almost have a clean base to start with

**Pull request merged**